system-tests/tests/smoke/cre/por_price_provider.go

```diff
@@ -243,6 +243,9 @@ func (f *FakePriceProvider) NextPrice(feedID string, price *big.Int, elapsed tim
 
 	if !f.priceAlreadyFound(cleanFeedID, price) {
 		f.testLogger.Info().Msgf("Feed %s updated after %s - price set, price=%s", cleanFeedID, elapsed, price)
+		if f.actualPrices == nil {
+			f.actualPrices = make(map[string][]*big.Int)
```

Comment:
maybe initialize the map, when provider is created?

Reply:
I'm not using the constructor, so I'm adding the nil check